app/controllers/charge_controller.js

```diff
@@ -254,21 +254,21 @@ module.exports = {
                 break
               case 500:
                 logging.failedChargePost(409)
```

Comment:
think this line and similar ones below are missing `req` as an argument? Unless I'm missing something

Reply:
@mrlumbu you are absolutely right! Did some bad rebasing and didn't push the changes :(